site/docs/master/aws-config.md

```diff
@@ -192,7 +198,8 @@ For more complex installation needs, use either the Helm chart, or add `--dry-ru
 It can be set up for Velero by creating a role that will have required permissions, and later by adding the permissions annotation on the velero deployment to define which role it should use internally.
 
 1. Create a Trust Policy document to allow the role being used for EC2 management & assume kube2iam role:
-```bash
+
+    ```bash
 cat > velero-trust-policy.json <<EOF
```

Comment:
This doesn't render properly in GH, I believe the content needs to be indented to line up with the backticks.